src/config/dynamic-theme-fixes.config

```diff
@@ -11109,6 +11109,21 @@ CSS
 
 ================================
 
+kaava.mafynetti.fi
+
+CSS
+.rich-text-editor img[src*="/math.svg"] {
+    background-color: white;
+}
+.rich-text-editor:focus img[src*="/math.svg"] {
+    background-color: white; 
+}
+.rich-text-editor.rich-text-focused img[src*="/math.svg"] {
+    background-color: white;
+}
```

Comment:
```suggestion
.rich-text-editor img[src*="/math.svg"] {
    background-color: white !important;
}
```

Reply:
Specifying `!important` is enough to ensure this would happen for the other states as well.

Reply:
Sure thing, I committed the suggestion, but I found another place with the exact same problem up in the toolbar that I also changed and now I don't know how to add it to this pull request. 

Here's a screenshot of what I changed:
![image](https://user-images.githubusercontent.com/44492823/217885913-bc57dcd4-0352-4806-9453-0317ecfe13f0.png)

And here's the CSS code for it:

.rich-text-editor-toolbar-equation {
    background-color: white;
    color: white;
}

This one looks even worse than the other one but It's functional.

Reply:
There we go, I didn't find it earlier as the website didn't come up in the code when I Ctrl + F:ed it. Seems like you have to do this inside the code window to find it.